test/revert.bats

```
@@ -343,8 +370,10 @@ test_revert_after_execute_master_failure() {
 
     # Execute should fail.
     local status=0
-    gpupgrade execute --verbose || status=$?
+    run gpupgrade execute --verbose
```

There's a concern that this will swallow the verbose output; if there's a failure below(say in revert), we won't see that output....maybe do:
```bash
echo "$output"
```
also?

---

Done.